src/Application/Components/Shop/KeyBoundShopItem.jsx

```diff
@@ -16,6 +18,10 @@ class KeyBoundShopItem extends React.Component {
 
   render() {
     const imgSrc = `images/${this.props.name}.png`;
+    let itemDetails = false;
```

Comment:
`{undefined}` and `{false}` are equivalent in the html block so u only need `let itemDetails;`